index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/base.css">
+    <link rel="stylesheet" href="./styles/index.css">
+</head>
+<body>
+    <header class="header row">
+        <nav class="header__nav container">
+            <div class="header__nav_logo cell">
+                <img src="./images/logo.svg" alt="logo">
+            </div>
+            <ul class="header__nav_list cell">
+                <li class="header__nav_list-item">The Team</li>
+                <li class="header__nav_list-item">Pricing</li>
+                <li class="header__nav_list-item">Features</li>
+            </ul>
+        </nav>
+        <section class="header__section container">
+            <div class="header__section_text cell">
+                <h1 class="header__section_text-title">Beautiful Free Nova template</h1>
+                <h4 class="header__section_text-description">A top notch premium quality template for your next web project.</h4>
+                <button class="header__section_text-button">Download FREE!</button>
+            </div>
+            <div class="header__section_image cell">
+                <img src="./images/screen.png" alt="screen" class="screen__image">
+        </section>
+    </header>
+    <main class="features row" id="Features">
+        <section class="features__section container">
+            <h2 class="features__section_title ">Features</h2>
+            <div class="features__section_box ">
+                <div class="features__section_box-feature">
```

Comment:
Mamy tuta listę cech, skoro tak to użyłbym `ul > li`, aby zmniejszyć ilość `div`